### PR TITLE
Increase the init memory size and numa config to avoid vm crash

### PR DESCRIPTION
**Pull request by jinqi7**
Signed-off-by: qijing <jinqi@redhat.com>

Increase the inital memory size and numa config to avoid crashing

### CODE REVIEW

**Comment by jinqi7** [comment]
# avocado run --vt-type libvirt libvirt_mem.positive_test.memory.hot.plug.max_slots.without_reboot
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 37ea07cafedc3a056ef94772edf22ab67b100dee
JOB LOG    : /root/avocado/job-results/job-2021-06-24T23.43-37ea07c/job.log
 (1/1) type_specific.io-github-autotest-libvirt.libvirt_mem.positive_test.memory.hot.plug.max_slots.without_reboot: PASS (241.43 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 242.31 s

File: libvirt/tests/cfg/libvirt_mem.cfg [hunk 1]
```diff
@@ -48,9 +48,12 @@
                     variants:
                         - default:
                         - max_slots:
+                            max_mem = 4194304
+                            current_mem = 4194304
```
Comment (lento-sun) [inline libvirt/tests/cfg/libvirt_mem.cfg:52]: If we do not add the 'max_mem = 4194304 current_mem = 4194304' here, this code branch will use 'max_mem = 1024000  current_mem = 1024000' just under the '- positive_test' branch. So my concern here if 1024000 is too small to start a vm (vm'll be crashed), will other cases under this 'postive_test' hit same issue?

Comment (jinqi7) [reply libvirt/tests/cfg/libvirt_mem.cfg:52]: Other cases  passed with using the original value 1024000.

**Review by smitterl — APPROVED** [review]
LGTM

**Review by lento-sun — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**